src/trace_capture/web/static/workspace.css

```
@@ -590,6 +590,218 @@ textarea:focus-visible {
+
+.worker-enrollment-result {
+  border-block-start: var(--border-width) solid var(--color-accent);
+  padding-block-start: var(--space-3);
+}
+
+.worker-enrollment-result__code code {
+  color: var(--color-accent);
+  flex: 1 1 14rem;
+  font-family: var(--font-mono-stack);
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)